show-build-logs.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
```

Comment:
I would either make this part of the Python scripts or move this into `ci` folder to make clear this is only useful in a particular context. 

E.g., if you have multiple logs in that folder this is going to be very verbose.

Reply:
I moved the new bash script `show-build-logs.sh` into a new `ci` directory.